src/main/java/frc/trigon/robot/subsystems/collector/placeholdercollector/PlaceholderCollectorConstants.java

```diff
@@ -0,0 +1,124 @@
+import frc.trigon.robot.subsystems.collector.CollectorConstants;
+
+public class PlaceholderCollectorConstants {
+    static final boolean FOC_ENABLED = true;
+    private static final int
+            COLLECTING_MOTOR_ID = 1,
+            ANGLE_MOTOR_ID = 2,
+            ENCODER_ID = 1;
+    private static final InvertedValue
+            COLLECTING_MOTOR_INVERTED = InvertedValue.CounterClockwise_Positive,
+            ANGLE_MOTOR_INVERTED = InvertedValue.CounterClockwise_Positive;
+    private static final NeutralModeValue
+            COLLECTING_MOTOR_NEUTRAL_MODE = NeutralModeValue.Coast,
+            ANGLE_MOTOR_NEUTRAL_MODE = NeutralModeValue.Brake;
+    private static final AbsoluteSensorRangeValue ENCODER_RANGE = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+    private static final SensorDirectionValue ENCODER_DIRECTION = SensorDirectionValue.CounterClockwise_Positive;
+    private static final double ENCODER_OFFSET = 0;
+    private static final FeedbackSensorSourceValue ENCODER_TYPE = FeedbackSensorSourceValue.FusedCANcoder;
+    private static final double
+            MOTION_MAGIC_VELOCITY = 200,
+            MOTION_MAGIC_ACCELERATION = 160,
+            MOTION_MAGIC_JERK = 1600;
+    private static final double
+            ANGLE_P = 0.1,
+            ANGLE_I = 0,
+            ANGLE_D = 0,
+            ANGLE_KA = 0,
+            ANGLE_KG = 0,
+            ANGLE_KS = 0,
+            ANGLE_KV = 0;
+    private static final CANcoder ENCODER = new CANcoder(ENCODER_ID, RobotConstants.CANIVORE_NAME);
+    static final TalonFX
+            COLLECTING_MOTOR = new TalonFX(COLLECTING_MOTOR_ID, RobotConstants.CANIVORE_NAME),
+            ANGLE_MOTOR = new TalonFX(ANGLE_MOTOR_ID, RobotConstants.CANIVORE_NAME);
+
+    static final StatusSignal<Double>
+            ANGLE_POSITION_SIGNAL = ENCODER.getPosition(),
+            ANGLE_VELOCITY_SIGNAL = ENCODER.getVelocity(),
+            ANGLE_MOTOR_CURRENT_SIGNAL = ANGLE_MOTOR.getStatorCurrent(),
+            ANGLE_MOTOR_VOLTAGE_SIGNAL = ANGLE_MOTOR.getMotorVoltage(),
+            ANGLE_MOTOR_PROFILED_SETPOINT_SIGNAL = ANGLE_MOTOR.getClosedLoopReference(),
+            COLLECTION_MOTOR_VELOCITY_SIGNAL = COLLECTING_MOTOR.getMotorVoltage(),
+            COLLECTION_MOTOR_CURRENT_SIGNAL = COLLECTING_MOTOR.getStatorCurrent(),
+            COLLECTION_MOTOR_VOLTAGE_SIGNAL = COLLECTING_MOTOR.getMotorVoltage();
+
+    static {
+        configureEncoder();
+        configureCollectingMotor();
+        configureAngleMotor();
+    }
+
+    private static void configureCollectingMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.Inverted = COLLECTING_MOTOR_INVERTED;
+        config.MotorOutput.NeutralMode = COLLECTING_MOTOR_NEUTRAL_MODE;
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+        config.Feedback.RotorToSensorRatio = CollectorConstants.COLLECTION_MOTOR_GEAR_RATIO;
+
```

Comment:
rotor to sensor ratio only works for a fused cancoder. You want to use SensorToMechanismRatio